### ISSUE TITLE
Cannot scroll up when looking at past messages

### ISSUE THREAD

**Issue by zkrx** [RocketChat/Rocket.Chat#31569]
### Description:

Use the search function or a link to go to a past message (e.g. from a year ago). Try to scroll up. Messages older than that old message should be fetched and displayed. Instead, rocketchat fetches messages older than the most recent message (e.g. from the previous hour). This is highly impractical as you need to go through the entire history if you want to see "very" old messages. Note that going down in the history (loading messages newer than that "very" old message) works without problem.

### Server Setup Information:

- Version of Rocket.Chat Server: 6.5.0
- Operating System: Linux
- Deployment Method: docker
- Number of Running Instances: 1
- NodeJS Version: v14.21.3
- MongoDB Version: 6.0.12

### Client Setup Information

- Desktop App or Browser Version: Desktop v3.9.11 (reproduced on browser, version unknown)
- Operating System: Linux

**Comment by SySagar** [comment]
Will look into this issue

**Comment by SySagar** [comment]
@zkrx I am able to reproduce this bug. Will raise a pr soon.

**Comment by zkrx** [comment]
I don't think you do. You could just write a first message with a special keyword (e.g. "debug"), then spam a few dozen/hundreds unrelated messages (so that only a few appears in the main window and the first "debug" message is burried in history). Then search for "debug" and you should be able to reproduce.

**Comment by zkrx** [comment]
@SySagar
> your would need to have a database of 1 year chat

I don't think you do. You could just spam garbage, then write a message with a special keyword (e.g. "debug"), then spam a few dozen/hundreds unrelated messages. That way, only a few messages appear in the main window and the special "debug" message is burried in history. Then search for "debug" and you should be able to reproduce locally.

**Comment by SySagar** [comment]
@zkrx althugh I did have went through this approach and was unable to reproduce so maybe dumping more data would work. I will give it an extra try adn will update you.
